main/appengine_config.py

```diff
@@ -6,6 +6,10 @@
 if os.environ.get('SERVER_SOFTWARE', '').startswith('Google App Engine'):
   sys.path.insert(0, 'lib.zip')
 else:
+  if os.name == 'nt':
+    os.name = None
+    sys.platform = ''
```

Comment:
Verified on mine.. it works like this.. thanks!

Maybe we should file an issue on click for that matter.. WDYT?